editor/modes/visual-editor/block-drop-zone.js

```diff
@@ -0,0 +1,51 @@
+/**
+ * External Dependencies
+ */
+import { connect } from 'react-redux';
+import { reduce, get, find } from 'lodash';
+
+/**
+ * WordPress dependencies
+ */
+import { DropZone } from '@wordpress/components';
+import { getBlockTypes } from '@wordpress/blocks';
+
+/**
+ * Internal dependencies
+ */
+import { insertBlocks } from '../../actions';
+
+function BlockDropZone( { index, ...props } ) {
+	const dropFiles = ( files, position ) => {
+		const transformation = reduce( getBlockTypes(), ( ret, blockType ) => {
```

Comment:
Ah, that's true. Alternatively it could be a `find` on the `map`ped block types, but not sure this really saves us much aside from some compactness, since it still needs to iterate over all block types.

```js
const transformation = find(
	getBlockTypes().reduce( ( result, blockType ) => (
		result.concat( get( blockType, 'transforms.from', [] )
	), [] ),
	( transform ) => transform.type === 'files' && transform.isMatch( files )
} );
```

I don't feel strongly either way.